py/gmaps_geocode.py

```diff
@@ -6,11 +6,14 @@
 import os
 import sys
 
-import googlemaps
+# import googlemaps
 import pandas as pd
 
 from datetime import datetime
 
+from urllib.parse import urlencode, quote
+import json
+import requests
```

Comment:
We probably want to leave all of the imports (i.e., add new imports but not delete old ones).